Source/Views/TableViews/SPTableView.m

```diff
@@ -294,6 +295,28 @@ - (void)rightMouseDown:(NSEvent *)event
 	}
 }
 
+- (void)mouseDown:(NSEvent *)event
+{
+    if (event.modifierFlags & NSEventModifierFlagOption && event.modifierFlags & NSEventModifierFlagCommand){
+        if ([[[[self delegate] class] description] isEqualToString:@"SPTablesList"]) {
+            if ([[self delegate] respondsToSelector:@selector(openTableInNewTab:withRowIndex:)]) {
+                if ([[self delegate] respondsToSelector:@selector(tableView:shouldSelectRow:)]) {
+                    if ([[self delegate] tableView:self shouldSelectRow:[self rowAtPoint:[self convertPoint:[event locationInWindow] fromView:nil]]]) {
+                        [(SPTablesList*)[self delegate] openTableInNewTab:self withRowIndex:[self rowAtPoint:[self convertPoint:[event locationInWindow] fromView:nil]]];
+                        // The tab bar will be empty if there is no mousedown event, I have no idea to fix it, so I add the below code to cheat the system to active the window...
+                        // If anyone knowns how to fix it, please do it.
+                        // Thanks.
+                        [super rightMouseDown:event];
```

Comment:
// The tab bar will be empty if there is no mousedown event, I have no idea to fix it, so I add the below code to cheat the system to active the window...
                        // If anyone knowns how to fix it, please do it.
                        // Thanks.

Reply:
Can you elaborate a little more here? Perhaps we need to check if there really is an event (?)

Reply:
I didn't know what happened here :(

If there is no `super` method, a strange bug will appear here.